fantasy_football_app/templates/fantasy_football_app/rules.html

```diff
@@ -0,0 +1,96 @@
+<!-- edit-entry.html -->
+{% load static %}
+{% load widget_tweaks %}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Edit Your Entry - Fantasy Football Playoff Challenge</title>
```

Comment:
Rename this to "Rules - Fantasy Football Playoff Challenge" or else the tab will say "Edit Your Entry"